tox.ini

```diff
@@ -19,7 +19,8 @@ whitelist_externals =
     mkdir
     rm
 deps =
-    django111: Django>=1.11,<2
+    django22: -rrequirements/django.txt
+    django30: Django>3.0
```

Comment:
change the dependency specification for Django 3 to `django30: Django>=3.0,<3.1` for consistency